admin/services/check-start.service/check-start.service.js

```diff
@@ -146,7 +167,7 @@ checkStartService.prepareCheck2 = async function (
   if (prepareCheckServiceEnabled) {
     prepareCheckService.prepareChecks(pupilChecks)
```

Comment:
This is ultimately an async operation...

Reply:
as discussed, please add the await in this PR